docs/volumes.md

```diff
@@ -3,17 +3,17 @@ This document describes the current state of Volumes in kubernetes.  Familiarity
 
 A Volume is a directory, possibly with some data in it, which is accessible to a Container. Kubernetes Volumes are similar to but not the same as [Docker Volumes](https://docs.docker.com/userguide/dockervolumes/).
 
-A Pod specifies which Volumes its containers need in its [ContainerManifest](https://developers.google.com/compute/docs/containers/container_vms#container_manifest) property.
+A Pod specifies which Volumes its containers need in its [spec.volumes](http://kubernetes.io/third_party/swagger-ui/#!/v1beta3/createPod) property.
 
 A process in a Container sees a filesystem view composed from two sources: a single Docker image and zero or more Volumes.  A [Docker image](https://docs.docker.com/userguide/dockerimages/) is at the root of the file hierarchy.  Any Volumes are mounted at points on the Docker image;  Volumes do not mount on other Volumes and do not have hard links to other Volumes.  Each container in the Pod independently specifies where on its image to mount each Volume.  This is specified a VolumeMounts property.
 
 ## Resources
 
 The storage media (Disk, SSD, or memory) of a volume is determined by the media of the filesystem holding the kubelet root dir (typically `/var/lib/kubelet`).
-There is no limit on how much space an EmptyDir or PersistentDir volume can consume, and no isolation between containers or between pods.
+There is no limit on how much space an EmptyDir or HostDir volume can consume, and no isolation between containers or between pods.
 
-In the future, we expect that a Volume will be able to request a certain amount of space using a [resource](./resources.md) specification,
-and to select the type of media to use, for clusters that have several media types.
+Currently, only a Persistent Volume is able to request for a certain amount of space using a resource specification ([example](https://github.com/GoogleCloudPlatform/kubernetes/tree/master/examples/persistent-volumes)).
```

Comment:
This is a bit wrong.  It's not about persistent volumes but about the specific volume types that are supported as persistent volumes.  I would just leave it as it was, maybe tweak "a Volume will be able to" to "EmptyDir Volumes will be able to"...

It might be worth writing a short section on persistent volumes.